frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -189,6 +200,41 @@ export class CatalogListPage extends React.Component<CatalogListPageProps, Catal
     );
   }
 
+  normalizeHelmCharts(chartEntries: HelmChartEntries) {
+    const { namespace: currentNamespace = '' } = this.props;
+
+    return _.reduce(
+      chartEntries,
+      (normalizedCharts, charts) => {
+        charts.forEach((chart: HelmChart) => {
+          const tags = chart.keywords;
+          const name = _.startCase(chart.name);
+          const iconClass = getImageForIconClass('icon-catalog');
+          const tileImgUrl = chart.icon;
+          const tileIconClass = tileImgUrl ? null : iconClass;
+          const chartURL = _.get(chart, 'urls.0');
+
+          normalizedCharts.push({
+            obj: { ...chart, ...{ metadata: { uid: chart.digest } } },
+            kind: 'HelmChart',
+            tileName: `${name} v${chart.version}`,
+            tileIconClass,
+            tileImgUrl,
+            tileDescription: chart.description,
+            tags,
+            createLabel: 'Install Helm Chart',
+            tileProvider: _.get(chart, 'maintainers.0.name'),
+            documentationUrl: chart.home,
+            supportUrl: chart.home,
+            href: `/catalog/helm-install?chartURL=${chartURL}&preselected-ns=${currentNamespace}`,
```

Comment:
uri encode params? At least the `chartURL` since value comes from external source.

Reply:
Done.